quantum/process_keycode/process_leader.c

```diff
@@ -58,9 +52,8 @@ bool process_leader(uint16_t keycode, keyrecord_t *record) {
           keycode = keycode & 0xFF;
         }
 #endif // LEADER_KEY_STRICT_KEY_PROCESSING
-        if (leader_sequence_size < 5) {
-          leader_sequence[leader_sequence_size] = keycode;
-          leader_sequence_size++;
+        if (sizeof(leader_sequence) < 5) {
+          leader_sequence[sizeof(leader_sequence)] = keycode;
```

Comment:
OK, my bad. this should be leader_sequence_size not sizeof(leader_sequence). Move that to the 5 above.